---
template: resources.html
title: resources
---

# Resources



## PI research pages
* ### [Dr. Paul Morley](people/morley.md)
* ###  [Dr. Lee Pinnell](people/pinnell.md)
* ### [Dr. Robert Valeris-Chacin](people/valeris.md)
* ### [Dr. Matthew Scott ](people/scott.md)

## Micobial Ecology Group
* [Microbial Ecology Group (MEG) lab page](https://www.meglab.org/)
    * [MEGARes antimicrobial resistance gene database](https://www.meglab.org/megares/)
    * [AMR++ bioinformatic pipeline](https://www.meglab.org/amrplusplus/)


## Open Science - Sharing Your Research with the World:
MEG embraces the philosophy of open science to promote transparency, and improve repeatability and efficiency in research. In line with this principle, we are sharing the tools that we have developed through our research to facilitate metagenomic investigations of antimicrobial resistance using genomic sequencing and high-throughput computational analysis.

### MEGARes
The MEGARes V3.0 database contains sequence data for nearly 9,000 hand-curated antimicrobial resistance genes accompanied by an annotation structure that is optimized for use with high throughput sequencing. The acyclical annotation graph of MEGARes allows for accurate, count-based, hierarchical statistical analysis of resistance at the population level, much like microbiome analysis, and is also designed to be used as a training database for the creation of statistical classifiers.

### AMR++ Pipeline
AMR++ is a bioinformatic pipeline meant to aid in the analysis of raw sequencing reads to characterize the profile of antimicrobial resistance genes, or resistome. AMR++ was developed to work in conjuction with the the MEGARes database and its accompanying acyclical annotation structure that is optimized for use with high throughput sequencing and metagenomic analysis. AMR++ V3.0 adds a new feature for high-throughput verification of resistance-conferring SNPs in relevant gene accessions (ARGs).

