---
template: resources.html
title: resources
---

# Resources
## Open Science - Sharing Your Research with the World:
The VERO Research team embraces the philosophy of open science to promote transparency, and improve repeatability and efficiency in research. In line with this principle, we share tools that we have developed through our research to facilitate metagenomic investigations of microbial ecology, host genomics, and antimicrobial resistance using genomic sequencing and high-throughput computational analysis.

## GitHub Pages:
* [Microbial Ecology group GitHub](https://github.com/Microbial-Ecology-Group)
* [Dr. Pinnell GitHub](https://github.com/ljpinnell)
* Dr. Scott GitHub
* Dr. Valeris-Chacin GitHub
* [Dr. Doster GitHub](https://github.com/EnriqueDoster)

## Micobial Ecology Group:
* [Microbial Ecology Group (MEG) lab page](https://www.meglab.org/)
The Microbial Ecology Group (MEG) is an interdisciplinary group of scientists from multiple institutions that conduct collaborative research addressing the issues of microbial ecology in animal, public, and environmental health. Lead scientists for MEG hail from (alphabetically): Colorado State University, Texas A&M University, University of Florida, University of Minnesota, and West Texas A&M University.
  
## MEGARes:
* [MEGARes antimicrobial resistance gene database](https://www.meglab.org/megares/)
The MEGARes V3.0 database contains sequence data for nearly 9,000 hand-curated antimicrobial resistance genes accompanied by an annotation structure that is optimized for use with high throughput sequencing. The acyclical annotation graph of MEGARes allows for accurate, count-based, hierarchical statistical analysis of resistance at the population level, much like microbiome analysis, and is also designed to be used as a training database for the creation of statistical classifiers.

## AMR++ Pipeline:
* [AMR++ bioinformatic pipeline](https://www.meglab.org/amrplusplus/)
AMR++ is a bioinformatic pipeline meant to aid in the analysis of raw sequencing reads to characterize the profile of antimicrobial resistance genes, or resistome. AMR++ was developed to work in conjuction with the the MEGARes database and its accompanying acyclical annotation structure that is optimized for use with high throughput sequencing and metagenomic analysis. AMR++ V3.0 adds a new feature for high-throughput verification of resistance-conferring SNPs in relevant gene accessions (ARGs).

## Miscellaneous resources
* [Checkout this page with a random assortment of links we found to be useful.](resourceslinks.md)